apps/codeforafrica/src/components/ErrorHero/ErrorHero.js

```diff
@@ -0,0 +1,53 @@
+import { RichTypography, Section } from "@commons-ui/core";
+import React from "react";
+
+import TwoToneBackground from "@/codeforafrica/components/TwoToneBackground";
+
+const ErrorHero = React.forwardRef(function ErrorHero(props, ref) {
+  const { title, subtitle, sx, tags, ...other } = props;
+
+  return (
+    <TwoToneBackground
+      sx={{ py: { xs: 7.5, md: 10, lg: 12.5 } }}
+      {...other}
```

Comment:
```suggestion
      sx={{ py: { xs: 7.5, md: 10, lg: 12.5 }, ...other?.sx }}
```

Nothing else needs to be passed down.